### ISSUE TITLE
Message search results should show room name, not room ID

### ISSUE THREAD

**Issue by ajkessel** [mszpro/Nil#7]
When searching on message contents, the results indicate the room by room ID, not room name. The user is not going to recognize any room by ID--the room name should be displayed in search results instead.

**Comment by mszpro** [comment]
I agree. The UI should fetch the room info based on the room id. Implementing this.

Also, I am thinking that tapping on the message should technically locate you to that location in the timeline.

@ajkessel

**Comment by mszpro** [comment]
Tracked internally at https://radar.magicat.dev/show_bug.cgi?id=7
